docs/Ruleset.md

```diff
@@ -109,6 +111,8 @@ V076 | MAJOR | Interface method parameter typing removed
 V077 | MINOR | Interface method parameter default added
 V078 | MAJOR | Interface method parameter default removed
 V079 | MAJOR | Interface method parameter default value changed
+V151 | PATCH | Interface method renamed (case only)
```

Comment:
This should be split into 3 rules, one for each visibility level (as it is done for all other rules even if they are all the same value).

Reply:
Interfaces can only have public methods

Reply:
True. In this case it should only be for the public method visibility. (the message was copy/pasted in each case :wink:)